### PR TITLE
Add support for \ instead of just λ in the parser

### PR DESCRIPTION
**Pull request by naoeosavio**

### CODE REVIEW

File: src/term/syntax.rs [hunk 1]
```diff
@@ -52,7 +52,9 @@ fn parse_text<'a>(code: &'a Str, text: &Str) -> Result<&'a Str, String> {
 // Parses a term, returns the remaining code and the term.
 pub fn parse_term<'a>(code: &'a Str, ctx: &mut Context<'a>, idx: &mut u32, definitions: &mut HashMap<DefinitionName, Term>) -> (&'a Str, Term) {
   let code = skip_whitespace(code);
+  // println!("{:?}",code);
```
Comment (Boscop) [inline src/term/syntax.rs]: Please remove this line :)

File: src/term/syntax.rs [hunk 2]
```diff
@@ -52,7 +52,9 @@ fn parse_text<'a>(code: &'a Str, text: &Str) -> Result<&'a Str, String> {
...
   match code[0] {
+
```
Comment (Boscop) [inline src/term/syntax.rs]: Please also remove this line :)

**Review by Boscop — CHANGES_REQUESTED** [review]
Good job :)
Just please remove those two lines.

File: src/term/syntax.rs [hunk 3]
```diff
@@ -86,6 +86,16 @@ pub fn parse_term<'a>(code: &'a Str, ctx: &mut Context<'a>, idx: &mut u32, defin
       (code, Lam { nam, typ, bod })
     },
     // Untyped Abstraction: `λvar body`
+    b'\\' => {
```
Comment (Boscop) [inline src/term/syntax.rs]: Please move this comment down to the branch where it was before (`b'\xce' if code[1] == b'\xbb' => {`) and write here: 
```
// Untyped Abstraction: `\var body`
```

File: src/term/syntax.rs [hunk 4]
```diff
@@ -86,6 +86,16 @@ pub fn parse_term<'a>(code: &'a Str, ctx: &mut Context<'a>, idx: &mut u32, defin
       (code, Lam { nam, typ, bod })
```
Comment (Boscop) [inline src/term/syntax.rs:86]: I forgot about this, please also add a `b'\\' => {` here, for the typed abstraction:
```rs
// Typed Abstraction: `\(var: Type) body`
b'\\' => {
...
```

**Review by Boscop — CHANGES_REQUESTED** [review]
Please also add these for completeness :)

File: src/term/syntax.rs (hunk 4 above)
Comment (naoeosavio) [reply src/term/syntax.rs:86]: I had asked you about it

**Review by Boscop — APPROVED** [review]
Thanks, great work :)

**Pull request merged**